Makefile.dep

```diff
@@ -364,6 +364,7 @@ endif
 
 ifneq (,$(filter isrpipe,$(USEMODULE)))
   USEMODULE += tsrb
+  USEMODULE += xtimer
```

Comment:
Is this necessary here?

Reply:
Yes it is.  sys\isrpipe relies on both tsrb and xtimer in the source code.  Compiling any of the examples what leverage uart_stdio (which depends on isrpipe) with thie hifive1 board and fe310 cpu fails because the xtimer modules are not linked in.

Reply:
Hm, that dependency sucks, as it is not needed everywhere. Maybe we need an isrpipe_timeout module or something...

Anyhow, this shouldn't cause linker problems unless the xtimer using fuctions are actually used. Are you using the same garbage collecting options for ld?

Reply:
"-ffunction-sections" was missing from the cpu flags.